### ISSUE TITLE
cut a new release

### ISSUE THREAD

**Issue by josharian** [crawshaw/sqlite#77]
I just hit https://github.com/crawshaw/sqlite/issues/58. The approach mentioned in #65 works, but it'd be better to just cut a new release instead of instructing people to use the master branch.

**Comment by AdamSLevy** [comment]
@josharian I have not seen any activity on this repo for quite some time. I believe the maintainer is MIA for now.

I have been maintaining a fork with a number of improvements. I recommend using it via a replace directive in your go.mod. 

This of course doesn't resolve this issue that you are raising. In fact it complicates the go.mod file and importing this package, but for the trouble, by using my fork you will be using the latest version of sqlite3 (3.30.1), and have all of my bug fixes and improvements to the go package. It is backwards compatible so it is a simple drop in replacement.

My fork is here:

https://github.com/AdamSLevy/sqlite

Continue importing this package as usual using "crawshaw.io/sqlite" but add this replace directive to your `go.mod` file and run `go mod tidy`.

```
replace crawshaw.io/sqlite => github.com/AdamSLevy/sqlite master
```

My hope is that my patches will eventually get merged here, but until then I am keeping my fork up to date with sqlite and any bugs reported here. I use my fork extensively in a project so it is well road-tested.

**Comment by crawshaw** [comment]
Hi @AdamSLevy! Sorry for the long absence, apparently I have a big chunk of github notifications turned off. (Only discovered this because @josharian pinged me out of band.)

I took a cursory look at your changes and they look good. Would you like me to add you as a collaborator on this repository?

**Comment by AdamSLevy** [comment]
Hi @crawshaw! I actually saw your talk about this package at Go Northwest 2018 which is what initially turned me onto this package.

Thanks for taking a look at my changes! I would be happy to be added as a collaborator on this repo. If you have no objections with any of my patches, I would just go ahead and push my master branch to this repo which includes all of my open PRs.

**Comment by crawshaw** [comment]
I added you as a collaborator.

Read through your commits, they look good! (Any small stuff we can deal with going forward.) Please go ahead and push your master branch. Given the quality of your work, please don't wait on me in the future, consider yourself empowered to commit anything.

**Comment by AdamSLevy** [comment]
I released a v0.2.0
